jobs/process_paid_filings/devops/vaults.gcp.env

```diff
@@ -0,0 +1,18 @@
+DATABASE_USERNAME="op://database/$APP_ENV/business-ar-db/DATABASE_USERNAME"
+DATABASE_PASSWORD="op://database/$APP_ENV/business-ar-db/DATABASE_PASSWORD"
+DATABASE_NAME="op://database/$APP_ENV/business-ar-db/DATABASE_NAME"
+DATABASE_UNIX_SOCKET="op://database/$APP_ENV/business-ar-db/DATABASE_UNIX_SOCKET"
```

Comment:
DATABASE_UNIX_SOCKET  is not used in the codebase. You probably need to copy this from AR Reminder job: https://github.com/bcgov/business-ar/blob/a67f02e3dd49feeb5e3457f3f068208c72f04aa1/jobs/ar-reminder/src/ar_reminder/config.py#L72

Reply:
Probably a good idea to check config.py to see which variables are read there, so that there is 1-to-1 corresponding between this files and read variables.

Reply:
I think this job didn't really establish connection to the database, probably I should just delete those lines in vaults.gcp.env?